modules/data/src/main/java/io/fluo/webindex/data/fluo/PageObserver.java

```diff
@@ -59,16 +59,22 @@ public void process(TransactionBase tx, Bytes row, Column col) throws Exception
     if (!curJson.isEmpty()) {
       Page curPage = gson.fromJson(curJson, Page.class);
       curLinks = curPage.getOutboundLinks();
-    } else {
-      Long score = ttx.get().row(row).col(FluoConstants.PAGE_SCORE_COL).toLong(0);
-      Long newScore = score + 1;
-      ttx.mutate().row(row).col(FluoConstants.PAGE_SCORE_COL).set(newScore);
+    }
+
+    if (curJson.isEmpty() && !nextJson.equals("delete")) {
+      Long incount = ttx.get().row(row).col(FluoConstants.PAGE_INCOUNT_COL).toLong();
+      if (incount == null) {
+        ttx.mutate().row(row).col(FluoConstants.PAGE_INCOUNT_COL).set(new Long(0));
+      }
+      FluoUtil.updateDomainPageCount(ttx, row, 1);
```

Comment:
Next time we talk, want to discuss using two Collision free maps to compute a page's inbound links and a domains inbound links.